cmd/yggdrasil/main.go

```diff
@@ -2,57 +2,26 @@ package main
 
 import (
 	"context"
-	"crypto/ed25519"
-	"encoding/hex"
 	"encoding/json"
 	"flag"
 	"fmt"
-	"net"
 	"os"
 	"os/signal"
-	"regexp"
-	"strings"
 	"syscall"
 
 	"github.com/gologme/log"
 	gsyslog "github.com/hashicorp/go-syslog"
 	"github.com/hjson/hjson-go/v4"
 	"github.com/kardianos/minwinsvc"
 
-	"github.com/yggdrasil-network/yggdrasil-go/src/address"
-	"github.com/yggdrasil-network/yggdrasil-go/src/admin"
 	"github.com/yggdrasil-network/yggdrasil-go/src/config"
-	"github.com/yggdrasil-network/yggdrasil-go/src/ipv6rwc"
-
-	"github.com/yggdrasil-network/yggdrasil-go/src/core"
-	"github.com/yggdrasil-network/yggdrasil-go/src/multicast"
-	"github.com/yggdrasil-network/yggdrasil-go/src/tun"
+	"github.com/yggdrasil-network/yggdrasil-go/src/setup"
```

Comment:
anecdote: this split is something that i've wanted for a while for my own reasons for a while.

Reply:
@neilalexander You told me on Matrix that setup split is not needed. @majestrate what's your reason?

Reply:
it seems to make initializing the core much simpler. i also attempted to do a netstack version of ygg a while back and that lack of split was a burden for me.